src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -122,9 +112,8 @@ CEvaluationResult* CCrossValidation::evaluate()
 
 	/* perform all the x-val runs */
 	SG_DEBUG("starting %d runs of cross-validation\n", m_num_runs)
-	for (index_t i=0; i <m_num_runs; ++i)
+	for (index_t i = 0; i < m_num_runs; i++)
```

Comment:
I would say it's clang-format fault. Anyway, no, there are no particular reasons for the postifx operator instead of the prefix one.